geonode/base/models.py

```diff
@@ -968,54 +952,61 @@ def group_name(self):
 
     @property
     def bbox(self):
-        """BBOX is in the format: [x0,x1,y0,y1]."""
-        return [
-            self.bbox_x0,
-            self.bbox_x1,
-            self.bbox_y0,
-            self.bbox_y1,
-            self.srid]
+        """BBOX is in the format: [x0, x1, y0, y1, srid]."""
+        if str(self.bbox_polygon.srid) not in self.srid:
+            self.bbox_polygon.transform(self.srid)
+
+        bbox = self.bbox_helper
+        return [bbox.xmin, bbox.xmax, bbox.ymin, bbox.ymax, self.srid]
 
     @property
     def ll_bbox(self):
-        """BBOX is in the format: [x0,x1,y0,y1]."""
-        from geonode.utils import bbox_to_projection
-        llbbox = self.bbox[0:4]
-        if self.srid and 'EPSG:' in self.srid:
-            try:
-                llbbox = bbox_to_projection([float(coord) for coord in llbbox] + [self.srid, ],
-                                            target_srid=4326)
-            except Exception:
-                pass
-        return [
-            llbbox[0],  # x0
-            llbbox[1],  # x1
-            llbbox[2],  # y0
-            llbbox[3],  # y1
-            self.srid]
+        """BBOX is in the format [x0, x1, y0, y1, "EPSG:srid"]. Provides backwards
+        compatibility after transition to polygons."""
+        bbox = self.bbox_polygon
+        if bbox.srid != 4326:
+            bbox = bbox.transform(4326, clone=True)
 
-    @property
-    def ll_bbox_string(self):
```

Comment:
Please restore this property. This might be used by client libraries to retrieve the lon-lat bbox representation.
Removing this might break backward compatibility.